arm-software/linux/README.md

```diff
@@ -303,3 +303,11 @@ $ armclang -fopenmp -o example example.o `pkg-config armpl-dynamic-lp64-omp --li
 $ chrpath -l ./example
 ./example: RUNPATH=$HOME/armpl/armpl_24.10_flang-new/lib/pkgconfig/../../lib:$HOME/atfl/lib/clang/20/lib/aarch64-unknown-linux-gnu:$HOME/atfl/bin/../lib/aarch64-unknown-linux-gnu
 ```
+
+## Providing feedback and reporting issues
+
+Please raise an issue via [Github issues](https://github.com/arm/arm-toolchain/issues).
```

Comment:
Do we want users to prefix [ATFL] for ATfL issues?

Reply:
We have labels https://github.com/arm/arm-toolchain/labels for ATfE and ATfL.

I would not require it to be set, but I can add a hint both here and in ATfE version of README.

Also, Arm Toolchain variant was added to the "bug" issue template https://github.com/arm/arm-toolchain/pull/305 to help clarify.